src/components/Configuration/Configuration.scss

```diff
@@ -0,0 +1,10 @@
+.shadow {
+  overflow: visible;
+  border-radius: 4px;
+  box-shadow: -1px 1px 10px 0 rgba(255, 255, 255, 0.44);
+}
+
+.inline {
+  padding-right: 15px !important;
```

Comment:
why important tags? maybe just add more specificity? or are we overriding inline styles here

Reply:
CSS need reworking, it was to add more specificity